res/translations/ru.json

```diff
@@ -277,5 +277,19 @@
         "few": "{{.trackCount}} композиции добавлены в список воспроизведения",
         "many": "{{.trackCount}} композиций добавлены в список воспроизведения",
         "other": "{{.trackCount}} композиций добавлены в список воспроизведения"
+    },
+
+    "{{.trackCount}} tracks": {
```

Comment:
Can you add these new translation templates to the other translation files as well? Without that, the localization lookup will fail for other languages. You can just make the "few"/"many"/"other" all the same, using whatever translation for "tracks" is in that localization. (Other translators can then update later if the few/many distinction is relevant for their language)

Reply:
Oh, sorry it looks like this may work as written because of your fallback logic. Shame on me for not testing before commenting! :)